lib/mutant/mutator/node/literal/regex.rb

```diff
@@ -31,6 +32,45 @@ def dispatch
             emit_type(s(:str, NULL_REGEXP_SOURCE), options)
           end
 
+          # Mutate regexp body
+          #
+          # @note will only mutate parts of regexp body if the
+          # body is composed of only strings. Regular expressions
+          # with interpolation are skipped
```

Comment:
I think mutant could recurse into the interpolation with ease, and these would also be very interesting, as more likely to reside inside subjects that actually get mutated.

And I'm fine to do this in the next iteration.